src/amber/controller/render.cr

```diff
@@ -2,6 +2,24 @@ module Amber::Controller
   module Render
     LAYOUT = "application.slang"
 
+    protected def render_text(body, status_code = 200)
```

Comment:
@elorest what do you think of dropping the `render_` and just using `text`, `json`, `html`?

Reply:
I think it would be better if their names are verbs, since verbs are more readable and nouns may collide with local variables.

Reply:
yup, your right.  It reads better.

Reply:
@drujensen Which ones. I just thought that text, json, html might be more likely to collide with something else.

Reply:
I don't know how you all feel about it, but I like the idea of just using the normal `render` command with named params like Rails does.

```crystal
def index
    render json: { key: "Value }
end
```

Reply:
IMHO render is already complicated enough. Also text and json aren't truly rendered, but rather just copied over into the response without changes. As these helper methods currently  are they don't have to be macro's or complicate the already complex workings of the render macro.

Reply:
Do you think there's a better word like `set_text, set_html, set_json`?

Reply:
The verb could be `respond_with` instead of render? Maybe something like:

```crystal
def respond_with(json: Hash(String | Symbol, Amber::Router::ParamsType))
    ...
end

def respond_with(text: String)
    ...
end

def respond_with(html: XML | String)
    ...
end
```

Or something like that.

Reply:
I'm ok with that. @drujensen @eliasjpr @fridgerator WDYT?

Reply:
I'm ok with that. @drujensen @eliasjpr @fridgerator WDYT?

Reply:
@watzon that looks like a good solution.  

Should it support the ability to detect the `Accept` header and allow you to define multiple response types?
```
respond_with(
  json: {"hello" => "world"},
  text: "hello world", 
  html: render("index.slang")
)
```

Reply:
I like the signature above is very intuitive. Crystal supports defining method with external names so the above can be written as

```crystal
respond with: { "json" }

def respond(with type)
  ...
end
```

See External Names here https://crystal-lang.org/docs/syntax_and_semantics/default_values_named_arguments_splats_tuples_and_overloading.html

Reply:
Yeah it's on my crystal lang coffee mug lol.

@eliasjpr In your code block above there's nothing that defines which type it's responding with though.

I think that this makes more sense.

```cr
respond_with(
  json: {"hello" => "world"},
  text: "hello world", 
  html: render("index.slang")
)
```